profiles/kentik_snmp/fortinet/fortinet-fortigate.yml

```diff
@@ -348,3 +349,27 @@ metrics:
       #Lease Usage of the DHCP server.
       - OID: 1.3.6.1.4.1.12356.101.23.2.1.1.2
         name: fgDhcpLeaseUsage
+
+  - MIB: FORTINET-FORTIGATE-MIB
+    table:
+      name: fgHaStatsTable
+      OID: 1.3.6.1.4.1.12356.101.13.2.1
+    symbols:
+      # Current HA Sync status
+      - name: fgHaStatsSyncStatus 
+        OID: 1.3.6.1.4.1.12356.101.13.2.1.1.12
+        enum:
+          unsynchronized: 0
+          synchronized: 1
+      # Serial number of primary during the last sync attempt (successful or not)
+      - name: fgHaStatsPrimarySerial
+        OID: 1.3.6.1.4.1.12356.101.13.2.1.1.16
```

Comment:
@PolovicJ - thanks for the submission. 

This OID should probably also be under the `metric_tags` just below since it's a string value.

If you expect it to be volatile (i.e.; have the value change on a regular basis) you could leave it here and add the [conversion: to_one](https://github.com/kentik/snmp-profiles/blob/69a02b1c70c7ab9336bd23748efaa6462c2d2faf/profiles/kentik_snmp/_template.yml#L98) setting to force it to poll on your standard interval. The result will be a more recent value, at the cost of collecting more samples.

Both techniques will give you the result you're seeking, just on different intervals. Up to you.

Reply:
Thank you for the review @thezackm !

`fgHaStatsPrimarySerial` can change after cluster failover. Even though I don't expect it to happen very often, I believe it could serve its purpose during troubleshooting. I added the suggested conversion.